terraform_docs.sh

```diff
@@ -132,8 +132,8 @@ terraform_docs() {
       --path-to-file)
         text_file=$value
         ;;
-      --add-to-exiting-file)
-        add_to_exiting=$value
+      --add-to-existing-file|--add-to-exiting-file) # Typo left for compatibility. Will be removed in 2.0.0.
```

Comment:
@antonbabenko just said that we don't need such greater backward compatibility, so...
 
```suggestion
      --add-to-existing-file)
```

Reply:
> @antonbabenko just said that we don't need such greater backward compatibility, so...

IMHO, it's not about **we** or **need**. It's about being polite and careful about consumers by adhering to `semver`. As such we could have had this "quirk" removed within future `2.0.0` release (alongside announcing this). IMHO.

Reply:
There are 3 users (including Maxym) of this argument (GitHub code search) :) In general, I agree that we need to keep compatibility.